packages/react/cypress/component/auto/form/ShadcnAutoEnumInput.cy.tsx

```diff
@@ -0,0 +1,395 @@
+import React from "react";
+import { elements } from "../../../../spec/auto/shadcn-defaults/index.js";
+import { apiTriggerOnly } from "../../../../spec/auto/support/Triggers.js";
+import { getStadiumRecord } from "../../../../spec/auto/support/stadiumModel.js";
+import { makeAutocomponents } from "../../../../src/auto/shadcn/index.js";
+import { api } from "../../../support/api.js";
+import { ShadcnWrapper } from "../../../support/auto.js";
+
+const ShadcnAutoForm = makeAutocomponents(elements).AutoForm;
+
+describe("ShadcnAutoEnumInput", () => {
```

Comment:
Are we able to use shared suites here that test that this works roughly the same as polaris? I feel like its gonna be a real pain in the ass for the next person to deduplicate

Reply:
The element targets are quite different, so it might not be possible to use a shared suite . Although maybe if we switched away from targeting things with id and used either `data-cy="submit"`

Reply:
@airhorns I had a similar case in `AutoRolesInput` where we used a shared suite both I target elemets differently per suite . would you prefer that going forward?

` it("can render a role input, make a selection, and submit the role key", () => {
 
    if (name === SUITE_NAMES.POLARIS) {
      polarisAutoRoleInput();
    }

    if (name === SUITE_NAMES.SHADCN) {
      shadcnAutoRoleInput();
    }


    submit("Widget");
  });
});`